### ISSUE TITLE
Power not being used to calc fitness

### ISSUE THREAD

**Issue by trevori77** [thomaschampagne/elevate#709]
### Step 1: Are you in the right place?

  * Your problem may have been already covered in the FAQ. Did you check https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions?
  * Your problem or feature request may already exists in project issues section. Did you check https://github.com/thomaschampagne/elevate/issues?

### Step 2: Describe your environment

  * Plugin version: _____
  * Chrome/Opera/Chromium version: _____
  * OS version: _____
  
#### Steps to reproduce:

  1. _____
  2. _____
  3. _____
  
#### Actual Behavior:

  * What happened?
  * Do you have javascript errors logs in browser console?

  > Press F12 to see developer console and javascript logs

  ```javascript
  Copy and paste your console logs here.
  ```
  
#### Expected Behavior:

  * What did you expect to happen?
  
#### Media attachement:

>You can upload/attach any images to this ticket.

#### Providing your Elevate synced athlete activities can be usefull to solve the problem

> To export your activities click on "sync button" (top right and corner), then "Backup activities". You can zip the file and upload it here.

**Comment by trevori77** [comment]
Elevate - 6.8.0 Stable
Chrome - 69.0.3497.100 (Official Build) (64-bit)
OS - Win7 Ent SP1 64bit

All my rides in past two years have been done with power, however, Elevate is not calculating using power data, only HR. A different computer seems to work perfectly fine. The results between the two are markedly different. I have turned the 'use power data

I have uninstalled and reinstalled the plugin. I have ensured that the correct toggles are 'on' for using power and resynced activities
